internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -408,7 +408,11 @@ func (md *machineDeployment) updateMachineByReplace(ctx context.Context, e *mach
 
 	newMachineRaw, err := md.flapsClient.Launch(ctx, *e.launchInput)
 	if err != nil {
-		return err
+		if strings.Contains(err.Error(), "could not reserve resource for machine") {
+			return errors.New("The region you're trying to deploy is likely at capacity. Consider deploying to a new region with fly deploy --region <region> or trying again in a few hours\n")
+		} else {
+			return err
+		}
```

Comment:
oh wait, the place for this is https://github.com/superfly/flyctl/blob/965aba6d8e011302f48a039e632a5575e20653f6/internal/command/deploy/machines_deploymachinesapp.go#L355-L360